docs/benchmarks/defocus_gradient_analysis.ipynb

```diff
@@ -13,8 +13,8 @@
     "import numpy as np\n",
     "from scipy.spatial.distance import cdist\n",
     "from scipy.optimize import curve_fit\n",
-    "from scipy.stats import t\n",
-    "sns.set(style='ticks', context='notebook')"
+    "\n",
```

Comment:
Do all the double quotes also need to be escaped?

Reply:
Nvm, they are inside double quotes

Reply:
yeah, the double quotes and \n is how an ipynb looks like outside of jupyter